comms/DiskAttribute.cpp

```diff
@@ -11,6 +11,7 @@
 #include "UnicodeFile.h"
 #include "md5.hpp"
 #include "ModuleHelper.h"
+#include "../wlib/PreferenceDlg.h"
```

Comment:
Comms should not have a dependency on wlib

Reply:
No longer needed